driver/compenv.ml

```diff
@@ -358,7 +368,9 @@ let read_one_param ppf position name v =
         ccobjs := Misc.rev_split_words v @ !ccobjs
     end
 
-  | "ccopts" ->
+  | "ccopt"
+  | "ccopts"
+    ->
```

Comment:
Looks like an unrelated change that should be in another PR.

Reply:
Is it really necessary to open a new PR ? This change is only to be consistent with the names of arguments.